sbncode/CAFMaker/FillReco.cxx

```diff
@@ -16,6 +16,50 @@ namespace caf
            && slice.primary.size() > 0; // must have primary tracks/showers
   }
 
+  void FillStubVars(const sbn::Stub &stub,
+                    const art::Ptr<recob::PFParticle> stubpfp,
+                    caf::SRStub &srstub,
+                    bool allowEmpty) {
+
+    // allowEmpty does not (yet) matter here
```

Comment:
Another (better?) way to silence the compiler is to write `bool /*allowEmpty*/` in the parameter list

Reply:
I think this as an equivalent approach is ok -- it's mostly just there to be explicit.